index/accessibility-statement.njk

```diff
@@ -10,7 +10,6 @@
 
     <ul class="govuk-list govuk-list--bullet">
         <li>the 'Claim criminal injuries compensation' service, available at <a href="https://claim-criminal-injuries-compensation.service.justice.gov.uk/" class="govuk-link">https://claim-criminal-injuries-compensation.service.justice.gov.uk</a></li>
```

Comment:
should this be changed from a (now) single bullet point to a single sentence instead @Julieallan @GrahamBaird1 ?

Reply:
@armoj I would remove the bullet point and add the content as a continuation of 'This page only.....

Reply:
updated to be a single sentence